docs/mllib-ensembles.md

```diff
@@ -506,7 +506,7 @@ JavaRDD<LabeledPoint> testData = splits[1];
 
 // Train a GradientBoostedTrees model.
 //  The defaultParams for Classification use LogLoss by default.
-BoostingStrategy boostingStrategy = BoostingStrategy.defaultParams("Classification");
+BoostingStrategy boostingStrategy = BoostingStrategy.defaultParams(Algo.Classification);
```

Comment:
Seems that I am a little careless about the docs of Java version. I will update it.